feat: add phrases crud endpoints

ismtabo opened the pull request:
Updated openapi doc.

Created temporal error page for phrase routes.

Resolves #14

ismtabo committed:
feat: add phrases crud endpoints

Updated openapi doc.

Created temporal error page for phrase routes.

ismtabo committed:
chore: fix format

Canario0 commented on src/routes/(auth)/phrases/[phraseId]/+server.ts:
```diff
@@ -0,0 +1,58 @@
+  if (params.phraseId == null) {
+    throw error(404);
+  }
+  const phraseId = BigInt(params.phraseId);
```
**nitpick (blocking):** Conversion trigger an exception.

[According to the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) this conversion may trigger an exception when provided a non numeric char, making the next check not necessary. Same for all of this conversions.

Canario0 reviewed (changes requested):
We are close to finish :smile: but we have to work on the conversions and maybe we could move the api endpoints to /api/*

ismtabo committed:
refactor: move api endpoints to own route

Added matcher to phrase id parameter.

Fixed the conversion of route parameter to number.

ismtabo committed:
chore: fix format

ismtabo committed:
feat: add missing param matcher

ismtabo committed:
Merge remote-tracking branch 'origin/main' into feature/add-phrases-crud-endpoints
